p4-16/psa/psa.p4

```diff
@@ -55,17 +54,19 @@ typedef bit<unspecified> PacketLength_t;
 typedef bit<unspecified> EgressInstance_t;
 typedef bit<unspecified> Timestamp_t;
 
+const   PortId_t         PORT_RECIRCULATE = unspecified;
```

Comment:
New definition needed if we want this value to be used as the ingress_port at the beginning of ingress, for recirculated packets, as I think Vladimir suggested.